source/features/clean-conversation-headers.css

```diff
@@ -5,3 +5,36 @@
 .rgh-clean-conversation-headers-non-default-branch a {
 	color: var(--color-state-hover-primary-text, #fff);
 }
+
+.rgh-clean-conversation-header {
```

Comment:
Like in the now-removed comment, it'd be good to know which words this will end up hiding. This applies to most of the code here where the comments existed.